src/app/clients/add/formSections/NumberChildrenCard.tsx

```diff
@@ -99,7 +100,7 @@ const NumberChildrenCard: React.FC<CardProps> = ({
                                     ["Don't Know", "-1"],
                                 ]}
                                 listTitle="Age"
-                                defaultValue="-1"
+                                defaultValue={child.age!.toString()}
```

Comment:
Just got a but lost - you have defined Person object to have number or null but you said it must be not null? So what is the initialized age for child then? Is it still -1
(And a question of pure curiosity, why is age string not integer?)

Reply:
age is null only for adults, but is a number for children 
initialized age for a "new" child is -1 and if it is fetched from the database, then it's set to whatever was fetched
age is a string only in this case (for the dropdown) because the dataInput component restricts the type of the value to string

Reply:
Mmm I see!